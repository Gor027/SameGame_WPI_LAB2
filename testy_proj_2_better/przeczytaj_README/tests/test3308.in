H:::  H::H H  : 
HHH:    :H:H:::H HH 
H::HHH::H:  H: HH :H: : H:  :HH :H: : :H:HH : ::  :H:   H :  HHH
    HHH:H  HH  HH  H :HH :HH H:: :: H
HHH  :: : H::H  :::H:H:H ::HH::H:  H : :HH  H:H  
 ::H:::H:::H:  :H  HHHH :HH: HH:   :HH::: :H :  H::HH:   HH H::  HH H : :H:
 H: :H :: :: :HHHHHHH:HH ::H H:H:    H::H H:: :HH::: H:::
 HH H   H:H H : : :HH::HH ::HH: H::HH: H HH: :H:H H::: :HH
H  HH
  HH :: : : H H H:H: ::
  HHH H  ::: H:HHHHH:H  HHHH:::H:H:HH   H:: :HH: 
:: :HH  HHHHH :  ::::   : H : : HH: H  H 
HH   :H: :H:: : : H ::  HH HH:
:: HH:::HH   :HH  ::: ::    :H HH H::  ::::::: H:H 
:H H :H: H:H ::H:H  : HH: :::H: ::HHHH : H   H  H  ::   :H: :  :    H::H: ::H
HHH  :::H :  :::H::HH: ::H:HHH::H:H H:::: H ::
 :: :H:H::H: :H: :H   :HHHH  : :H:: :  H:H:H H : H:H :H: H:H:: : H:: HHH
::  HHHH:H H:: : HHH    HH  H H:: 
:H::: :  :::::H:  H : H:HH  H  :::HHH: HH ::H :H:
