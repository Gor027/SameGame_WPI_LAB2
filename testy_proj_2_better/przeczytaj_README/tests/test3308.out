      H :         H
 :   H: :H:H  :   :
 H   HH HHHH  :H  :H:        H  :
 :  H:::H::H::HHH HH: : H:  :HH :   :  H
HH::H:::::::H::HHH:HH H ::H H:::: : H  :
H:H::H::::HHHHH:::HHH::HH::HHH:HHH: H  :: H
HH:H::HH:::::::HHHHHHHHHHHHH:HH:::: H  :H :  :
HHHHHHH:HH:::H:HH::H:HHH:::::H:HH::HH: H:HH :: :
H::HH:::::HHH::H:HH::::HHH:HH::HHH::HH HHHHHH::HH  H
:H:HH:H:HHH:H:HHH::H:::::HH::::H:HHH:H :HH:::HH:H :H
H:HHH:::HH:HH:::HH:::::H:HHH:H::H:HHH:::::::::::H:::::  HH:
:HH:H:::H:::::::::H::HH:HH:HHH::H:H:HH::::H::HHH::::HH ::HH HH H     :
:HHH:HHH:HHH::H:::HHH::HHH::HH:::::H:::H:H:H::H:HHH:::::HH: ::H:HH H::H: :
HHHHHH:H::::::H:HHHH::H:HHHHH:H:::HHH::HHH::HH:H:H:H::H::H:H::::H::HHHH:H::H
